document-generation/igce-document.ts

```diff
@@ -36,13 +36,13 @@ export async function generateIGCEDocument(
     fundingDoc = payload.fundingDocument;
     switch (fundingDoc.fundingType) {
       case FundingType.FS_FORM:
-        fundingDocumentNumber = `Order Number: ${fundingDoc.orderNumber} and GT&C Number: ${fundingDoc.gtcNumber}`;
+        fundingDocumentNumber = `Order Number: ${fundingDoc.orderNumber}`;
```

Comment:
Is this supposed to be "Order Number" or "Funding Document Number" -- the reference images on Jira only show Funding Document Number.